libflux/src/core/parser/mod.rs

```diff
@@ -532,6 +532,78 @@ impl Parser {
             id,
         }))
     }
+    #[cfg(test)]
+    fn parse_type_expression(&mut self) -> TypeExpression {
+        let monotype = self.parse_monotype(); // monotype
+
+        // let t = self.peek();
+        // if t.tok == TOK_IDENT {
+        //     t = self.expect(TOK_INDENT); // where
+        //     if t.lit != "where" {
+        //         // handle error if where was not found
+        //         self.errs.push(format!(
+        //             "expected \"where\" but found : {}",
+        //             t.lit
+        //         ));
+        //     }
+        //
+        //     let constraints = self.parse_constraints();
+        //     // return the typeexpression
+        //     Expression::TypeExpression {
+        //         Monotype: monotype,
+        //         Constraints: constraints
+        //     }
+        // }
+        // return the typeexpression
```

Comment:
Can you remove the commented out code that's not being used? These comments won't show up in the diff next time, and I want to make sure we don't forget to remove them.